docs/proposals/last-operation-description.md

```diff
@@ -0,0 +1,97 @@
+# Last Operation Description
+
+## Introduction
+As per the OpenService Broker API spec, the last operation response can contain
+a description as well as a status: [last operation response](https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md#body).
+This can provide useful information about what is happening during a broker action along with details of the overall progress.
+UI implementors could make valuable use of this information to provide an indicator of progress and provide feedback while long running
+actions are in progress. An example of a possible description is shown below:
+```
+60%: succesfully created realm in keycloak
+```
+
+## Problem Description
+Currently we are only making use of the status field: 
+[last operation in broker.go](https://github.com/openshift/ansible-service-broker/blob/master/pkg/broker/broker.go#L1311)
+The difficulty is around how we get that detail back from the pod.
+Without the description, we limit the feedback that can be provided to the user for actions performed against the service catalog.
+
+## Expectations
+- The last_operation demonstrates APB progress.
+- No restrictions or requirements for user to demonstrate APB progress.
+- No guarantee that last_operation shows every operation.
+- The final_operation is the last_operation gathered before the APB is deleted.
+
+
+## Terms
+
+**Last Operation:**  the most recent operation an apb performed 
+
+**Final operation:** the operation that indicates the apb is 100% complete
```

Comment:
~~o~~Operation

Reply:
I'd also recommend lining this up with the expectations, `**Final Operation:** the last operation before APB completed.